lib/LWP/Authen/Basic.pm

```diff
@@ -11,6 +11,10 @@ sub auth_header {
     return "Basic " . MIME::Base64::encode("$user:$pass", "");
 }
 
+sub reauth_requested {
```

Comment:
Can we make these new methods private, please?

```sub _reauth_requested```

Reply:
Seems fine to me

Reply:
@Frank071 are you still interested in making this change?

Reply:
Yes, but tell me what to do... I am not that much into Git...